app/frontend/controllers/organisations_controller.js

```diff
@@ -0,0 +1,27 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  updateProfitStatusOptions (event) {
+    const profitStatusSelect = document.getElementById('organisation-profit-status-field')
+    if (!profitStatusSelect) return
+
+    const localAuthorityOption = profitStatusSelect.querySelector('option[value="local_authority"]')
+    const nonProfitOption = profitStatusSelect.querySelector('option[value="non_profit"]')
+    const profitOption = profitStatusSelect.querySelector('option[value="profit"]')
+    const providerType = event.target.value
+
+    profitStatusSelect.disabled = false
+    localAuthorityOption.hidden = false
+    nonProfitOption.hidden = false
+    profitOption.hidden = false
+
+    if (providerType === 'LA') {
+      profitStatusSelect.value = 'local_authority'
+      nonProfitOption.hidden = true
+      profitOption.hidden = true
+    } else if (providerType === 'PRP') {
+      profitStatusSelect.value = ''
+      localAuthorityOption.hidden = true
+    }
```

Comment:
This means that any time the provider type changes we're clearing this. I think that's fine since there are only two provider types, but can you think of any issues that might cause?

Reply:
I wasn't able to break it. considering these types come from existing org structure feels unlikely to need to add a new one in the near future